projectile.el

```diff
@@ -1949,7 +1949,8 @@ With FLEX-MATCHING, match any file that contains the base name of current file"
          (candidates
           (cl-sort (copy-sequence candidates)
                    (lambda (file _)
-                     (let ((candidate-dirname (file-name-nondirectory (directory-file-name (file-name-directory file)))))
+                     (let ((candidate-dirname (file-name-nondirectory
```

Comment:
What file is passed here when this breaks? Just the filename without a dir at all?

Reply:
Yes, just the filename. This seems to happen for candidate files in the project root directory.